pepper-apis/parent-pom.xml

```diff
@@ -424,7 +424,7 @@
         <dependency>
             <groupId>org.testcontainers</groupId>
             <artifactId>mysql</artifactId>
-            <version>1.12.3</version>
+            <version>1.17.6</version>
```

Comment:
Any particular reason to settle on this version?  I see that 1.18.1 was released on 5/11/2023.